src/index.js

```diff
@@ -3,6 +3,18 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
+import {Provider} from 'react-redux';
```

Comment:
it seems you didn't apply prettier :joy:

Reply:
help me with that